pkg/schema.go

```diff
@@ -187,6 +188,11 @@ func processComment(schema *Schema, comment string) (isRequired bool) {
 				schema.Items = &Schema{
 					Type: value,
 				}
+			case "itemEnum":
+				if schema.Items == nil {
```

Comment:
Is this check really needed?

Reply:
I think this check is required as a safety check to ensure that the Items field of the Schema struct is not nil before setting the Enum field on it. I have introduced another use case where only the `itemEnum` key is defined. Without this check the new test case fail due to nil pointer reference. However I have tested a version without the check and ran it against a value like this:

```
test: [1, 2] # @schema itemEnum:[1,2]
```

It is rendering the `values.schema.json` file as expected as the `Schema` is being initialized before the `processComment` function gets called. Should I remove it and remove the new test case too?

Thanks.